.github/workflows/mybinder-build.yml

```diff
@@ -0,0 +1,21 @@
+name: Trigger MyBinder build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    #tags:
+    #  - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Once the review is done, I will change this into executing only on merging so that we don't spam binder servers.